rspace/src/main/scala/coop/rchain/rspace/ISpace.scala

```diff
@@ -25,6 +27,13 @@ trait ISpace[C, P, A, R, K] {
 
   val branch: Branch
 
+  protected[this] val eventLog: SyncVar[Log] = {
```

Comment:
or: `SyncVarOps.create[Log](Seq.empty)`

Reply:
Oh nice.  We could make that implicit even fancier so that we could do `SyncVar.create`, but I'll save that work for another PR.